arcane/src/arcane/accelerator/MaterialVariableViews.h

```diff
@@ -134,6 +140,12 @@ class MatItemVariableScalarOutViewT
     return Accessor(this->m_value0[pmvi.valueIndex()]);
   }
 
+  //! Surcharge pour accéder à la valeure globale à partir du cell id
+  ARCCORE_HOST_DEVICE const DataType& operator[](Integer cid) const
```

Comment:
To be coherent with `VariableViews.h`, use `ItemIndexType` instead of `Integer` to index from global item. `ItemIndexType` is defined like that:

 `using ItemIndexType = typename ItemTraitsT<ItemType>::LocalIdType;`